README.md

```diff
@@ -16,3 +18,83 @@ _insert slide content here_
 ---
 
 _insert any explanations_ (will be in speaker notes)
+
+
+
+We are specifying these details to help the design team when they put the information into the actual PPT.
+
+First, you need to specify the type -- what slide template you would like to use. Our design team has made many good BitProject templates, and you can find all of them [here](https://docs.google.com/presentation/d/1F5_Qcf_0IduoNKzRUT4owPlE5t5Ab8utmGHUMwZCM_g/edit#slide=id.p). You can make a copy of it and see the details of all templates by adding empty slides. 
+
+
+
+For almost all the templates, they have a block for title. So simply put the title of the slide after "**Title:**". You can use the corresponding title of the card, or be more specific.
+
+
+
+Once you chose the template and finished the title, you should write the content of the slide. You need to specifically point out what content should be in which text/image box in the template. For example, there are 2 text boxes in the "small code snipet" template, so you need to specify what are in the upper text box, and what are in the lower text box. 
+
+
+
+Since the slides are to help people visualize the ideas better and be like a summary, you should simplify and extract the content from the cards instead of copying it. There should not be long sentences or paragraphs on the slides. 
```

Comment:
I think a specific example of how one can take more in-depth information from an activity and simplify/summarize it for a slide would help the reader understand the expectations